src/base/password_log.cpp

```diff
@@ -0,0 +1,895 @@
+/*
+ * Copyright 2008 Search Solution Corporation
+ * Copyright 2016 CUBRID Corporation
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+
+/*
+ * password_log.c -
+ */
+
+#ident "$Id$"
+
+#include <stdio.h>
+#include <stdarg.h>
+#include <stdlib.h>
+#include <time.h>
+#include <string.h>
+#include <errno.h>
+#if defined(WINDOWS)
+#include <sys/timeb.h>
+#include <process.h>
+#include <io.h>
+#else
+#include <unistd.h>
+#include <sys/time.h>
+#include <sys/timeb.h>
+#include <signal.h>
+#endif
+#include <assert.h>
+
+#include "porting.h"
+#include "cas_common.h"
+#include "password_log.h"
+//#include "parse_tree.h"
+//#include "system_parameter.h"
+//#include "environment_variable.h"
+//#include "broker_config.h"
+//#include "util_func.h"
+
+//extern PARSER_CONTEXT *this_parser;
+//==========================================================================
+
+static unsigned char isSpace[0x100] =
```

Comment:
trivial comment.
In CUBRID, C code, not Java code, we generally use hyphens rather than Camel Case notation.
isSpace => is_space

Reply:
OK, Thank you.